t/macros/tableau.t

```diff
@@ -43,25 +47,34 @@ unless (%context && $context{current}) {
 	Context();        # Initialize context (for persistent mod_perl)
 }
 
-sub WARN_MESSAGE {
-	warn("WARN MESSAGE: ", @_);
-}
-
-sub DEBUG_MESSAGE {
-	warn("DEBUG MESSAGE: ", @_);
-}
 Context("Matrix");
 
 Context()->flags->set(
 	zeroLevel    => 1E-5,
 	zeroLevelTol => 1E-5
 );
 
+
 my $A = Real(.0000005);
 my $B = Real(0);
 
-is($A, $B, "test zeroLevel tolerance");
-ok($A == $B, "test zeroLevel tolerance with ok");
+subtest 'test zeroLevel tolerance' => sub {
+	is($A->value, within($B->value, $B->getFlag('zeroLevel')), 'test zeroLevel tolerance');
+	ok($A == $B, 'test zeroLevel tolerance with ok');
+
+	my $real_object = object {
+		prop isa => 'Value::Real';
```

Comment:
I realized that the reason some of these tests are failing is because you used `isa`.  `isa` is not in the version of Test2::Suite that is included in the Ubuntu repositories (version 0.000129).

In any case, this should be using `blessed` and not `isa` as that is a more specific and accurate test (and is in version 0.000129).  `isa` only tells you if the given object is an instance of the given class (any class derived from the given class will pass the test).  However, `blessed` tells you more.  It tells you that the object is specifically blessed as that class, not just derived from.